README.md

```diff
@@ -297,11 +297,13 @@ The above is equivalent to the following `package.json` entry:
 
 These options are available:
 
-* **testLocator** - [Default: `"test/lib/**/*.js"`] - the file glob teenytest
-should use to search for tests
+* **testLocator** - [Default: `"test/lib/**/*.js"`] - one or more globs which
+teenytest should use to search for tests. May be a string or an array of strings
+* **example** - [Default: `[]`] - one or more global name filters to be applied
+to all files matched by `testLocator`
 * **helper** - [Default: `"test/helper.js"`] - the location of your global test
 helper file
-* **timeout** - [Default: `5000`] - the maximum timeout (in milliseconds) for any
+* **asyncTimeout** - [Default: `5000`] - the maximum timeout (in milliseconds) for any
```

Comment:
@searls I changed this because the option in the source looks to be `asyncTimeout` but if I'm missing something please let me know and I'll change it back.
https://github.com/testdouble/teenytest/blob/master/lib/configure/defaults.js#L7